qiskit_sphinx_theme/__init__.py

```diff
@@ -25,6 +26,26 @@ def _get_theme_absolute_path(folder_name: str) -> str:
     return str(path.resolve())
 
 
+def remove_thebe_if_not_needed(app, pagename, templatename, context, doctree):
+    """
+    Remove files that jupyter-sphinx incorrectly tries to add.
+    
+    See https://github.com/Qiskit/qiskit_sphinx_theme/issues/291 for more context. 
+    """
+    # jupyter-sphinx might be not installed. If so, skip this function.
+    try:
+        from jupyter_sphinx.thebelab import ThebeButtonNode
+    except ImportError:
+        return 
+
+    if not doctree  or doctree.traverse(ThebeButtonNode):    
+        thebe_js_files = ["_static/sphinx-thebe.js", "_static/thebelab-helper.js", "https://unpkg.com/thebelab@latest/lib/index.js"]
+        context["script_files"] = [js_file for js_file in context["script_files"] if js_file not in thebe_js_files]
+
+        thebe_css_files = ['_static/thebelab.css', '_static/sphinx-thebe.css']
+        context["css_files"] = [css_file for css_file in context["css_files"] if css_file not in thebe_css_files]
```

Comment:
You need to early return if the condition is True. 

```suggestion
    if not doctree or doctree.traverse(ThebeButtonNode):
        return

    thebe_js_files = ["_static/sphinx-thebe.js", "_static/thebelab-helper.js", "https://unpkg.com/thebelab@latest/lib/index.js"]
    context["script_files"] = [js_file for js_file in context["script_files"] if js_file not in thebe_js_files]

    thebe_css_files = ['_static/thebelab.css', '_static/sphinx-thebe.css']
    context["css_files"] = [css_file for css_file in context["css_files"] if css_file not in thebe_css_files]
```